.travis.yml

```diff
@@ -11,4 +15,7 @@ node_js:
   - "node"
 
 script:
+  - if [[ $TRAVIS_OS_NAME == "windows" ]]; then
+      git config core.autocrlf false && git add --renormalize . && git reset --hard;
+    fi
```

Comment:
why not run eslint on linux only?

Reply:
Running eslint on linux only has simpler syntax,

``` yml
  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then
      npm run eslint;
    fi
```

https://github.com/hexojs/hexo-renderer-pug/blob/master/.travis.yml

Reply:
Updated.